README.md

```diff
@@ -6,7 +6,173 @@ Camunda Form Playground.
 
 ## Usage
 
-...
+Integrate the playground into your application:
+
+```javascript
+import { createCamundaFormPlayground } from 'camunda/form-playground';
+
+const formPlayground = await createCamundaFormPlayground({
+  container: document.querySelector('#container'),
+  data,
+  schema
+});
+
+// open preview
+formPlayground.open();
+
+// close preview
+formPlayground.collapse();
+
+// listen on editor events
+const formEditor = formPlayground.getEditor();
+
+formEditor.on('selection.changed', () => { ... });
+```
+
+## Styling
+
+For proper styling include the necessary stylesheets, and font used:
+
+```html
+<link href="https://fonts.googleapis.com/css2?family=IBM+Plex+Sans:ital,wght@0,400;0,600;1,400&display=swap" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css2?family=IBM+Plex+Mono&display=swap" rel="stylesheet">
+
+<link rel="stylesheet" href="https://unpkg.com/@camunda/form-playground@0.1.0/dist/assets/form-js.css">
+<link rel="stylesheet" href="https://unpkg.com/@camunda/form-playground@0.1.0/dist/assets/form-js-editor.css">
+<link rel="stylesheet" href="https://unpkg.com/@camunda/form-playground@0.1.0/dist/assets/dragula.css">
+<link rel="stylesheet" href="https://unpkg.com/@camunda/form-playground@0.1.0/dist/assets/properties-panel.css">
+<link rel="stylesheet" href="https://unpkg.com/@camunda/form-playground@0.1.0/dist/assets/camunda-form-playground.css">
+```
+
+## API
+
+### `createCamundaFormPlayground => Promise<Result, Error>`
+
+Create a fully rendered form playground with options `{ container?: HTMLElement, data: any, exporter?: { name: String, version: String }, layout?: any, schema: any }`.
+
+```javascript
+import { createFormPlayground } from '@camunda/form-playground';
+
+const formPlayground = await createFormPlayground({
+  container: document.querySelector('#form-playground'),
+  data,
+  schema
+});
+```
+
+
+### `CamundaFormPlayground`
+
+Create a new form playground with options `{ container?: HTMLElement, data: any, exporter?: { name: String, version: String }, layout?: any, schema: any }`.
+
+```javascript
+import { CamundaFormPlayground } from '@camunda/form-playground';
+
+const formPlayground = new CamundaFormPlayground({
+  container: document.querySelector('#form-playground'),
+  data,
+  schema
+});
+```
+
+### `CamundaFormPlayground#open(containers: String[]) => void`
+
+Open all or specific playground containers.
+
+```javascript
+// open all
+formPlayground.open();
+
+// open specific
+formPlayground.open([ 'form-preview' ]);
+```
+
+### `CamundaFormPlayground#collapse(containers: String[]) => void`
```

Comment:
```suggestion
### `CamundaFormPlayground#collapse(containers?: String[]) => void`
```

If it's optional, we should mark it as optional (also in the type definitions if applies).

Reply:
BTW the TS way would be to use lowercase `string`.

Reply:
Adjusted it via https://github.com/camunda/form-playground/pull/1/commits/2e84b44abea82a0c5ec06672ced203c34e69072c